monitoring/src/config.ts

```diff
@@ -0,0 +1,115 @@
+import { mainCheck as mainCheckAus } from './check-page/aus';
+import { mainCheck as mainCheckCCPA } from './check-page/ccpa';
+import { mainCheck as mainCheckTcfV2 } from './check-page/tcfv2';
+import { envAwsRegion, envJurisdiction, envStage } from './env';
+import type { Config } from './types';
+
+type JurisdictionOpt = string | undefined;
+type AwsRegionOpt = string | undefined;
+
+const decideJurisdiction = (
+	jurisdiction: JurisdictionOpt,
+	awsRegion: AwsRegionOpt,
+): string => {
+	jurisdiction;
+	awsRegion;
+	if (jurisdiction) {
+		return jurisdiction;
+	}
+	if (awsRegion === 'eu-west-1') {
+		return 'tcfv2';
+	}
+	if (awsRegion === 'us-west-1') {
+		return 'ccpa';
+	}
+	if (awsRegion === 'ap-southeast-2') {
+		return 'aus';
+	}
+	return 'tcfv2'; // default value
+};
+
+const ConfigTcfv2Prod: Config = {
+	stage: 'prod',
+	jurisdiction: 'tcfv2',
+	frontUrl: 'https://www.theguardian.com',
+	articleUrl:
+		'https://www.theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
+	iframeDomain: 'https://sourcepoint.theguardian.com',
+	checkFunction: mainCheckTcfV2,
+};
+
+const ConfigTcfv2Code: Config = {
+	stage: 'code',
+	jurisdiction: 'tcfv2',
+	frontUrl: 'https://m.code.dev-theguardian.com',
+	articleUrl:
+		'https://m.code.dev-theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
+	iframeDomain: 'https://cdn.privacy-mgmt.com',
+	checkFunction: mainCheckTcfV2,
+};
+
+const ConfigCCPAProd: Config = {
+	stage: 'prod',
+	jurisdiction: 'ccpa',
+	frontUrl: 'https://www.theguardian.com/us',
+	articleUrl:
+		'https://www.theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
+	iframeDomain: 'https://ccpa-notice.sp-prod.net',
+	checkFunction: mainCheckCCPA,
+};
+
+const ConfigCCPACode: Config = {
+	stage: 'code',
+	jurisdiction: 'ccpa',
+	frontUrl: 'https://m.code.dev-theguardian.com/us',
+	articleUrl:
+		'https://m.code.dev-theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
+	iframeDomain: 'https://cdn.privacy-mgmt.com',
+	checkFunction: mainCheckCCPA,
+};
+
+const ConfigAusProd: Config = {
+	stage: 'prod',
+	jurisdiction: 'aus',
+	frontUrl: 'https://www.theguardian.com/au',
+	articleUrl:
+		'https://www.theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
+	iframeDomain: 'https://ccpa-notice.sp-prod.net',
+	checkFunction: mainCheckAus,
+};
+
+const ConfigAusCode: Config = {
+	stage: 'code',
```

Comment:
The `STAGE` environment variables (at least in CODE) that are handed to the lambda are uppercase so the stage matching below does not work and the tests are run against the production environment instead.

Perhaps we need to think about how this parameter will be set when we call this lambda?

Reply:
I think this might be the case because I can see prod config urls being called this way in the eu-west-1 CODE deployment. As discussed with @shtukas, possibly there is another cause.